bool[10] region_flags
int32 s32_NumVehicles
geometry_msgs/PoseArray ar_PoseVehicles